### DISCUSSION TITLE
default rule not working when using :param

### DISCUSSION THREAD

**Discussion by Dannydai123** [mockoon/mockoon#915] [Help]
when the rule is using /content/:param1 (route parameter eqs. "xzy")  with a default rule,   if your request URL is /content/ without any parameter, the default rule is to match , but not match now.

dont know something wrong with it.
 
Danny

**Comment by 255kb** [comment]
I suppose you received a 404? 
This is normal. If you define a path containing a parameter in on of its section, like `/content/:param1`, calling the server with `/content/` is trying to call **another** route. It will not reach your route and trigger the rules. It will just try to call `/content` which may not exist, or is a completely different route.
